### ISSUE TITLE
Touchpad scrolling does not work only with Precision Touchpad drivers

### ISSUE THREAD

**Issue by anonimuos** [Open-Shell/Open-Shell-Menu#718]
Running the latest Open Shell on two PCs, one with Precision Touchpad drivers and the other with OEM-specific (Synaptics) drivers. On the Synaptics, touchpad two finger scrolling a long single column All Programs menu in Open Shell works perfectly.

However this is broken on PCs with Precision Touchpad. Is there a simple fix?

**Comment by bonzibudd** [comment]
It works fine for me on my laptop in submenus and the Programs list, but not in the Search list when searching folder directories. I don't think OSM references code for precision scrolling at all, so this might require many changes if it's not a simple bug.

**Comment by anonimuos** [comment]
Yup I don't think it's a bug. There are two entirely different drivers - the older OEM-specific drivers and the Precision Touchpad drivers. They may be exposing different APIs for scrolling. The original Classic Shell was abandoned in 2017 and never updated for Precision Touchpad drivers. I use another app called "Everything Toolbar" (a taskbar toolband for Voidtools Everything) and it has the same issue - touchpad scrolling doesn't work with PTP drivers, works once the older OEM-specific (Alps, Synaptics) drivers are installed (depending on the laptop model of course).

**Comment by Gaurav-ClassicShellTester** [comment]
@bonzibudd Do you have Precision Touchpad drivers on the laptop where two-finger touchpad scrolling works in long single-column submenus? (Not the 7 style All programs tree - there it works for me too, only in long scrolling menus with scroll buttons, it's broken). On a different laptop with proprietary Synaptics drivers, it works.

One way to check is to open the Windows 10 Touchpad settings page. If it says "Your PC has a precision touchpad." then it has those PTP drivers installed.
